lib/vdyp-forward/src/main/java/ca/bc/gov/nrs/vdyp/forward/ForwardProcessingEngine.java

```diff
@@ -660,6 +600,101 @@ private void grow(
 			return;
 	}
 
```

Comment:
The growSpecies method had a lot of parameters, the Change object collects some of them together.